### PR TITLE
Use shared partials

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

**Review by csutter — APPROVED** [review]
🗑️

File: app/views/publishers/vacancies/_publish_buttons.html.slim [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- if vacancy.draft? && vacancy.errors.none?
+  h3.govuk-heading-m = t("jobs.preview_listing.heading")
+  p.govuk-body = t("jobs.preview_listing.message")
+  = govuk_button_link_to t("buttons.preview_job_listing"), organisation_job_preview_path(vacancy.id), class: "preview-listing-gtm", id: "vacancy-review-preview"
+
+  h3.govuk-heading-m = t("jobs.submit_listing.heading", count: vacancy.publish_on.today? ? 0 : 1, date: vacancy.publish_on.to_s.strip)
```
Comment (david-mears-dfe) [inline app/views/publishers/vacancies/_publish_buttons.html.slim:6]: I can't see how the `vacancy.publish_on` nil bug is accounted for here...

File: app/views/publishers/vacancies/review.html.slim [hunk 2]
```diff
@@ -8,40 +8,18 @@
   .vacancy.govuk-grid-row
     .govuk-grid-column-two-thirds
       h2.govuk-heading-l = t("publishers.vacancies.steps.review_heading")
-      = render "shared/error_messages", errors: @vacancy.errors
+      = render "shared/error_messages", errors: vacancy.errors
 
       p.govuk-body-l
         - if @vacancy.publish_on.today?
```
Comment (david-mears-dfe) [inline app/views/publishers/vacancies/review.html.slim:14]: Nor here. And should this be `vacancy` not `@vacancy`?

File: app/views/publishers/vacancies/_publish_buttons.html.slim (hunk 1 above)
Comment (cesidio) [reply app/views/publishers/vacancies/_publish_buttons.html.slim:6]: it's addressed on first line:

`- if vacancy.draft? && vacancy.errors.none?`

**Pull request merged**